src/main/java/org/dcsa/ebl/model/TransportPlan.java

```diff
@@ -0,0 +1,4 @@
+package org.dcsa.ebl.model;
+
+public class TransportPlan {
+}
```

Comment:
Deliberately empty?

Reply:
Yes - I am not finished yet... Will have to finish transportPlan in a future PR